lib/features/digital_guide_view/presentation/digital_guide_view.dart

```diff
@@ -0,0 +1,118 @@
+import "package:auto_route/annotations.dart";
+import "package:fast_immutable_collections/fast_immutable_collections.dart";
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../config/ui_config.dart";
+import "../../../gen/assets.gen.dart";
+import "../../../utils/context_extensions.dart";
+import "../../../utils/determine_contact_icon.dart";
+import "../../../utils/phone_numbers.dart";
+import "../../../widgets/detail_views/contact_section.dart";
+import "../../../widgets/detail_views/detail_view_app_bar.dart";
+import "../../../widgets/report_change_button.dart";
+import "../data/models/digital_guide_response.dart";
+import "../data/repository/digital_guide_repository.dart";
+import "../data/repository/image_repository.dart";
+import "widgets/accessibility_button.dart";
+import "widgets/digital_guide_data_source_link.dart";
+import "widgets/digital_guide_features_section.dart";
+import "widgets/headlines_section.dart";
+
+@RoutePage()
+class DigitalGuideView extends ConsumerWidget {
+  const DigitalGuideView({
+    @PathParam("id") required this.id,
+  });
+
+  final int id;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncDigitalGuideData = ref.watch(getDigitalGuideDataProvider(id));
+    // question: Should the app bar appear during loading or when there's an error?
+    // Now it doesn't, neither does it appear on SKS menu screen
+    return asyncDigitalGuideData.when(
+      data: _DigitalGuideView.new,
+      // To-do nice error
+      error: (error, stackTrace) => Text("API error occured: $error"),
+      loading: () => const Scaffold(
+        body: Center(
```

Comment:
Everything else in the file we use shimmer loading as a way of indicating loading state. We should make it here as well, but now I will create separate task for it.